### PR TITLE
Graduate backendconfig to GA

### PR DESCRIPTION
**Pull request by skmatti**
PR highlights:
1. Generated code by running command `./hack/update-codegen.sh`
2. Update Backendconfig API to V1
3. Add backendconfig GA annotation

This PR does not update backendconfig version in end to end tests. That will be updated in a separate PR and merged after this is released.

### CODE REVIEW

**Comment by rramkumar1** [comment]
@skmatti In order for this to work, you need to add "v1" as a version in the CRD specification. 

Ref: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/

**Comment by rramkumar1** [comment]
Looks good to me. Would like someone else to sanity check this as well.

File: pkg/annotations/service.go [hunk 1]
```diff
@@ -270,3 +273,15 @@ func (svc *Service) GetBackendConfigs() (*BackendConfigs, error) {
 	}
 	return &configs, nil
 }
+
+// getBackendConfigAnnotation returns specified backendconfig annotation value.
+// Returns false if both beta and ga annotations are not specified.
+func (svc *Service) getBackendConfigAnnotation() (string, bool) {
```
Comment (freehan) [inline pkg/annotations/service.go]: add comment it prefer returning GA annotation first.

**Pull request merged**